docs/quickstart.rst

```diff
@@ -1,8 +1,58 @@
-Quickstart guide
+Quickstart Guide
 ################
 
+Prerequisite
+************
+
+In the latest release, Ape requires: 
+
+**Linux**
+* Python 3.7.X or later
+
+**Windows**: 
+
+#. Install Windows Subsystem Linux `(WSL) <https://docs.microsoft.com/en-us/windows/wsl/install>`_ 
+#. Choose Ubuntu 20.04 OR Any other Linux Distribution with Python 3.7.X or later
+
+Please make sure you are using Python 3.7.X or later.
+
+.. code-block:: bash
+    # check your python version
+    $ python3 --version
+
+
+
+
 Installation
 ************
+**Suggestion**: Create a virtual environment via ``virtualenv`` or ``venv`` .
+
+You may skip this creating a virtual environment if you know you don't require one for your use case. 
+
+via ``virtualenv`` `virtualenv <https://pypi.org/project/virtualenv/>`_
```

Comment:
It seems like this line and the next one (line 35) are very similar. Is that intentional? Can we remove one?

The ones that start with `via virtualenv`

Reply:
rewrote this